11-node-frameworks/classworks/Volodkova_Anna/index.js

```diff
@@ -0,0 +1,50 @@
+const {incCounter, schema} = require('./functions') ;
+const Router = require('koa-router')
+const koa = require('koa');
+const serve = require('koa-static');
+const session = require('koa-generic-session');
+const bodyParser = require('koa-bodyparser');
+
+const app = new koa();
+
+app.use(bodyParser());
+
+app.use(serve(`public`));
+
+app.use(session());
+const inc = incCounter(session);
+
+const router = new Router();
+
+router.post('/summer', (ctx) => {
+  inc();
+  const {error, value} = schema.validate(ctx.request.body);
+  if (!error) {
+    ctx.body = JSON.stringify(value);
+  } else {
+    ctx.body = "Not valid name or mark";
```

Comment:
Use JSON for each response, this string is invalid JSON. Also, send errors from Joi validator, that helps the user understand which errors occurred.